src/Tasks/Microsoft.NET.Build.Tasks/Microsoft.NET.Build.Tasks.csproj

```diff
@@ -52,8 +52,10 @@
     <PackageReference Include="Microsoft.Build" ExcludeAssets="Runtime" />
     <PackageReference Include="Microsoft.Build.Framework" ExcludeAssets="Runtime" />
     <PackageReference Include="Microsoft.Build.Utilities.Core" ExcludeAssets="Runtime" />
-    <PackageReference Include="Microsoft.Extensions.DependencyModel" />
-    <PackageReference Include="Microsoft.NET.HostModel" />
+    <PackageReference Include="Microsoft.Extensions.DependencyModel" VersionOverride="8.0.0-rc.1.23419.4"/>
```

Comment:
@marcpopMSFT - I see you hard coded the version in https://github.com/dotnet/sdk/commit/3cd48fd80f26c5a23cc2f0c69cba738764b149c0.  I am assuming this is a temporary workaround of sorts.  If that is the case, then an exception should be added to the [prebuilt baseline](https://github.com/dotnet/sdk/blob/main/eng/SourceBuildPrebuiltBaseline.xml)

```
    <!-- Add comment describing the temporary nature of these and link a tracking issue to remove. -->
    <UsagePattern IdentityGlob="Microsoft.NET.HostModel/8.0.0-rc.1.23419.4" />
    <UsagePattern IdentityGlob="System.Text.Json/8.0.0-rc.1.23419.4" />
    <UsagePattern IdentityGlob="Microsoft.Bcl.AsyncInterfaces/8.0.0-rc.1.23419.4" />
```

Reply:
correct, we were getting task exceptions because the 8.0 system.text.json wouldn't load so I had to add a hardcoded package reference in the task assembly and then add 3 more dependencies of that package.